src/components/CreateVmWizard/steps/Storage.js

```diff
@@ -260,10 +261,9 @@ class Storage extends React.Component {
               />
             </FormGroup>
             <span className={style['disk-size-edit-label']}>GiB</span>
-            <FieldLevelHelp
-              inline
-              content={msg.diskEditorSizeCreateHelp()}
-            />
+            <div>
```

Comment:
Is the wrapping `div` necessary?

Reply:
I found collisions between the css attributes of the cell and  the PF icon so the shortest way without create any regressions was to wrap i with div, if you have any better suggestion please let me know.